pkg/algorithm/constants.go

```diff
@@ -40,30 +40,30 @@ const (
 
 	// internal cell states
 
-	// No affinity group is using, acquiring or has acquired the cell.
+	// No affinity group is using, reserving, or has reserved the cell.
 	// A Free cell's priority must be freePriority.
 	cellFree CellState = "Free"
-	// An affinity group is using this cell, and no other group is acquiring it.
+	// An affinity group is using this cell, and no other group is reserving or has reserved it.
 	// A Used cell's priority is that of the group using the cell.
 	cellUsed CellState = "Used"
-	// An affinity group is using this cell, and another group is acquiring it.
-	// An Acquiring cell's priority is that of the group acquiring the cell. This means the scheduling algorithm
-	// will respect the acquiring group, i.e., a group with a non-higher priority cannot get this cell.
-	cellAcquiring CellState = "Acquiring"
-	// No affinity group is using this cell, and a group has acquired it.
-	// An Acquired cell's priority is that of the group that acquired the cell. This means the scheduling algorithm
-	// will respect the group that acquired the cell, i.e., a group with a non-higher priority cannot get this cell.
-	cellAcquired CellState = "Acquired"
+	// An affinity group is using this cell, and another group is reserving it.
+	// A BeingReserved cell's priority is that of the group reserving it. This means the scheduling algorithm
+	// will respect the reserving group, i.e., a group with a non-higher priority cannot get this cell.
+	cellBeingReserved CellState = "BeingReserved"
```

Comment:
>cellBeingReserved [](start = 1, length = 17)

Cell cannot reserve others. And align with cellUsed/cellReserved, just cellReserving is fine?